Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN cd /tmp && /tmp/install_ta-lib.sh && rm -r /tmp/*ta-lib*
 ENV LD_LIBRARY_PATH /usr/local/lib
 
 # Install dependencies
-COPY --chown=ftuser:ftuser requirements.txt requirements-hyperopt.txt /freqtrade/
+COPY --chown=ftuser:ftuser requirements.txt requirements-hyperopt.txt requirements-freqai.txt /freqtrade/
```

Comment:
the regular dockerfile is not supposed to contain freqAI requirements - that'll bloat the image too much for users who don't need this.
there is a `Dockerfile.freqai` - which is available in the docker folder and installs these requirements - freqAI users should therefore use the image `:develop_freqai` - as is also stated in the documentation.

Reply:
@richardjozsa any chance you could try this method so we can remove that change?

Reply:
assuming you build this locally (you'll have to anyway) - you can simply add (or uncomment) a "build" step in he compose file - pointing it to the dockerfile.freqai in the docker directory.

Reply:
It was me yeah, for testing in docker, it was easier in the test phase. I can make a dockerfile.freqai in the docker folder so the users can build it up from that.

Reply:
> I can make a dockerfile.freqai in the docker folder so the users can build it up from that.

That file does exist ... https://github.com/freqtrade/freqtrade/blob/develop/docker/Dockerfile.freqai